### DISCUSSION TITLE
osu!mania scroll acceleration

### DISCUSSION THREAD

**Discussion by Redageddon** [ppy/osu#19616] [Ideas]
I'm thinking it could work kind of like how scroll speeds could work. This would be used in addition to, not a replacement of, scroll speed. Default acceleration would be 0 (its ignored, and scroll speed just uses the base timing values). >0 values would have the notes come in at a slower speed, and accelerate up to the scroll speed time exactly at the judgement line. <0 values would have the notes come in at a faster speed, and accelerate down to the scroll speed time exactly at the judgement line. 
EX: this is what a -3x, 0x, 3x acceleration would look like (this is only for comparison I'm not saying each lane should have different accelerations)
![Accel](https://user-images.githubusercontent.com/53098956/183272596-d1b45636-32fb-4ce3-89e8-ac4aaa36e5d2.png)

**Comment by peppy** [comment]
Not sure if this would work better as a setting or a mod, but I think it could be an interesting one to have available.

**Reply by Redageddon** [reply]
Imo it would not be as good as a mod because it doesn't change hit timings. And it would also make sense to have it right next to scroll speed in settings, that could make it a bit more intuitive.

**Reply by eclecticfluff** [reply]
Wouldn’t it make most sense to make the approach different mod work in all game modes? At least they should probably have the same options, wherever the mania approach different ends up.

**Reply by Redageddon** [reply]
No.
std: it doesn't make much sense with approach rate, and is cancelled with hidden
taiko: it directly clashes with how taiko works, as taiko already has bpm independent notes, so acceleration on that would be terrible
catch: it could maybe work, but it seems like it would be a difficulty mod instead of a helper mod
